bugwarrior/db.py

```diff
@@ -25,8 +26,9 @@
 MARKUP = "(bw)"
 
 
-DOGPILE_CACHE_PATH = os.path.expanduser(
-    os.getenv('XDG_CACHE_HOME', '~/.cache') + '/dagd.dbm')
+PYVER = '%i.%i' % sys.version_info[:2]
+DOGPILE_CACHE_PATH = os.path.expanduser(''.join([
+    os.getenv('XDG_CACHE_HOME', '~/.cache'), '/dagd-py', PYVER, '.dbm']))
```

Comment:
Ah ha, the [python 3 docs](https://docs.python.org/3/library/dbm.html) point to this [third party lib](https://www.jcea.es/programacion/pybsddb.htm) which explains that the berkeley db interface was included in py2.3 through py2.7 but is not included in python3.  I'll see if just installing it will do as a workaround.